### PR TITLE
#56 - Added "Link and References" link in navbar

### LINKED ISSUE #56: Add links to Links and References section
The links are the same as the ones in the navigation

### PR DESCRIPTION
**Pull request by mtwn105**
This closes #56

### CODE REVIEW

**Review by bronzehedwick — APPROVED** [review]

File: browser/src/App.js [hunk 1]
```diff
@@ -240,7 +240,7 @@ function App() {
           The team at Astoria Digital is using the data from these request to
           create visualizations to show their process in real time.
         </p>
-        <h2>Links and References</h2>
+        <h2 id="links_and_references">Links and References</h2>
         <ul>
           <li>All Requests</li>
```
Comment (aniagaska) [inline browser/src/App.js:245]: Could you also add [All Requests](https://www.muckrock.com/foi/list/?projects=778) and [Materials and Articles ](https://www.muckrock.com/project/new-york-officer-misconduct-and-disciplinary-materials-778/)

Comment (aniagaska) [reply browser/src/App.js:245]: I now realize that my wording was not clear. The ul in All Requests and Links and References need to be hyperlinked, and they are the same links as the ones in the nav bar for reference!

Comment (ashryanbeats) [reply browser/src/App.js:245]: Let's create @aniagaska's asks as a secondary issue.

**Review by ashryanbeats — APPROVED** [review]

**Comment by ashryanbeats** [comment]
Thanks for the PR @mtwn105!

**Pull request merged**